iree/tools/BUILD

```diff
@@ -251,6 +259,7 @@ cc_library(
     name = "iree_translate_main",
     srcs = ["translate_main.cc"],
     deps = [
+        ":init_compiler_modules",
```

Comment:
I think this means you should be able to drop the dep on IREE_COMPILER_DIALECT_MODULES below and at that point, just inline the variable. This is nice anyway because dep variables confuse a lot of build file tooling

Reply:
Good point!